### PR TITLE
Ltclm zeitreihen releases

### PR DESCRIPTION
**Pull request by ltclm**
this pr modifies the timestamp aggregation within the releases service.
The timestamps are grouped by bgdi_order before they are merged together.

### CODE REVIEW

**Comment by ltclm** [comment]
[testlink releases](http://mf-chsdi3.dev.bgdi.ch/ltclm_zeitreihen_releases/rest/services/all/MapServer/ch.swisstopo.zeitreihen/releases?geometry=650000.0%2C170000.0&imageDisplay=500%2C600%2C96&geometryType=esriGeometryPoint&mapExtent=611399.9999999999%2C158650%2C690299.9999999999%2C198150;)
[testlink identify](http://mf-chsdi3.dev.bgdi.ch/ltclm_zeitreihen_releases/rest/services/all/MapServer/ch.swisstopo.zeitreihen/releases?geometry=650000.0%2C170000.0&imageDisplay=500%2C600%2C96&geometryType=esriGeometryPoint&mapExtent=611399.9999999999%2C158650%2C690299.9999999999%2C198150;)

File: chsdi/views/features.py [hunk 1]
```diff
@@ -401,15 +401,21 @@ def _get_features_for_filters(params, models, maxFeatures=None, where=None):
...
-                    bgdi_order = 0
-                    for feature in query:
-                        counter += 1
-                        if counter > 1:
-                            if bgdi_order < feature.bgdi_order:
-                                continue
-                        bgdi_order = feature.bgdi_order
-                        yield feature
+                    # standard identify show first bgdi_order only
+                    if maxFeatures == 201:
```
Comment (gjn) [inline chsdi/views/features.py]: This if could be put togehter with the previous if:

`if model.__bodId__ == 'ch.swisstopo.zeitreihen' and maxFeatures == 201:`

201 is magic number. Would prefer to have constant defined at top of file MAX_FEATURES = 201 that is then used verywhere.

**Comment by ltclm** [comment]
https://github.com/geoadmin/mf-chsdi3/issues/1639 can be closed when this is merged

**Comment by gjn** [comment]
I pushed forced a little fix (linting was not happy).

**Comment by gjn** [comment]
@ltclm Rolf was just in my office. I tested the realase and the print is perfect. He tested 3 regions with 3 different scales.

I'll merge. Thanks alot.

**Comment by loicgasser** [comment]
This heavily conflicts with https://github.com/geoadmin/mf-chsdi3/pull/1980 ...

**Pull request merged**